cmip6_downscaling/config/config.py

```diff
@@ -9,8 +9,10 @@
 
 # Azure --------------------------------------------------------------------
 connection_string = os.environ.get("AZURE_STORAGE_CONNECTION_STRING")
+CONNECTION_STRING = connection_string
 storage = Azure("prefect")
-intermediate_cache_store = CacheStore('az://flow-outputs/intermediate')
+intermediate_cache_path = 'az://flow-outputs/intermediate'
+intermediate_cache_store = CacheStore(intermediate_cache_path)
 results_cache_store = CacheStore('az://flow-outputs/results')
```

Comment:
nit: could the results cache be split into two lines to mirror the syntax of the intermediate cache path/store?

Reply:
yes! done